apps/ci/ci-install-modules.sh

```diff
@@ -9,3 +9,4 @@ git clone --depth=1 --branch=master https://github.com/azerothcore/mod-transmog.
 git clone --depth=1 --branch=master https://github.com/azerothcore/mod-npc-beastmaster.git modules/mod-npc-beastmaster
 git clone --depth=1 --branch=master https://github.com/azerothcore/mod-duel-reset.git modules/mod-duel-reset
 git clone --depth=1 --branch=master https://github.com/azerothcore/mod-premium modules/mod-premium
+git clone --depth=1 --branch=master https://github.com/Winfidonarleyan/mod-cfbg modules/mod-cfbg
```

Comment:
shall we use `https://github.com/azerothcore/mod-cfbg` instead ? I've just forked your repo. We usually put under `azerothcore/` the most important modules (and I believe your module is very important!)

Reply:
I change this, for only test pr with module.
I think we can do that.